app/models/likes.rb

```diff
@@ -0,0 +1,7 @@
+class Likes < ApplicationRecord
+  belongs_to :posts, :users
+
+  def update_likes_count
+    posts.likes_counter += 1
+  end
```

Comment:
## Optional
- [ ] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the #update_likes_counter method every time a new like has been saved to database, then make the `update_likes_counter` method private